helm/aragorn/values.yaml

```diff
@@ -116,11 +116,15 @@ redis:
   image:
     repository: redis
     tag: "7.0.12"
+  replicaCount: 2
   master:
     resources:
       limits:
-        cpu: 2
-        memory: 60Gi
+        cpu: 8
```

Comment:
why lower the memory than CI?

Reply:
If we give 60 the application goes into a pending state to create PVC, so I lower the memory to use 58G. If anything breaks the application will try to fix it.